pkg/events/parse_args.go

```diff
@@ -27,6 +39,28 @@ func ParseArgs(event *trace.Event) error {
 		}
 	}
 
+	if fdArg := GetArg(event, "fd"); fdArg != nil {
```

Comment:
A map lookup to each call to ParseArgs will slow down the pipeline for all cases when we have "option:parse-arguments". That is definitely not what we want. Maybe we should have another option "option:parse-arguments-fds" instead, that would only do the fds translations if explicitly requested (with a warning that translating file descriptors might cause slow dows in the pipeline).

Reply:
I agree. For sure `option:parse-arguments-fds` can be created.